code/modules/projectiles/guns/energy/kinetic_accelerator.dm

```diff
@@ -296,7 +297,7 @@
 
 /obj/item/borg/upgrade/modkit/proc/install(obj/item/gun/energy/kinetic_accelerator/KA, mob/user)
 	. = TRUE
-	if(src in KA.modkits) // Sanity check to prevent installing the same modkit twice thanks to occasional click/lag delays.
+	if(src in KA.modkits && KA.yeetdelay < world.time) // Sanity check to prevent installing the same modkit twice thanks to occasional click/lag delays.
```

Comment:
```suggestion
	if((src in KA.modkits) || KA.yeetdelay >= world.time) // Sanity check to prevent installing the same modkit twice thanks to occasional click/lag delays.
```